app/scripts/controllers/network/enums.js

```diff
@@ -39,6 +39,14 @@ export const NETWORK_TYPE_TO_ID_MAP = {
   [MAINNET]: { networkId: MAINNET_NETWORK_ID, chainId: MAINNET_CHAIN_ID },
 }
 
+export const NETWORK_TO_ID_MAP = {
+  [ROPSTEN_CHAIN_ID]: ROPSTEN_NETWORK_ID,
+  [RINKEBY_CHAIN_ID]: RINKEBY_NETWORK_ID,
+  [KOVAN_CHAIN_ID]: KOVAN_NETWORK_ID,
+  [GOERLI_CHAIN_ID]: GOERLI_NETWORK_ID,
+  [MAINNET_CHAIN_ID]: MAINNET_NETWORK_ID,
+}
```

Comment:
This isn't needed since they are the same numbers just string representations of each other